dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerAPIImpl.java

```diff
@@ -430,6 +441,12 @@ public Container save(Container container, List<ContainerStructure> containerStr
 			}
 		}
 
+		try {
+			String bla = null;
+		} catch (Exception e) {
+			e.getMessage();
+		}
+
 		// save the container-structure relationships , issue-2093
 		for (ContainerStructure cs : containerStructureList) {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Combine this loop with the one that starts on line 398. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3047)